dist/404.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>Page Not Found</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <style>
```

Comment:
Should the 404 page be self-contained, or should the styles go in a .css file? Also, should normalize.css not be included?

Reply:
Personally, i think this should stay self-contained for now. Until we actually have a page design for it.